src/Visithor/Compiler/Compiler.php

```diff
@@ -235,8 +235,8 @@ private function addVendorFiles(Phar $phar)
             ->name('*.php')
             ->exclude('Tests')
             ->in(realpath($vendorPath . 'symfony/'))
-            ->in(realpath($vendorPath . 'guzzlehttp/'))
-            ->in(realpath($vendorPath . 'react/'));
```

Comment:
That's probably something I forgot to add back. I was testing the StreamClient, so that the hhvm build would work.